wire-library/wire-runtime-swift/src/main/swift/AnyMessage.swift

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2020 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+/**
+ * Wire implementation of the `google.protobuf.Any` type. The `Any` type wraps an arbitrary
+ * protobuf message. Field of `Any` generated by Wire will be typed [AnyMessage]. [typeUrl] is
+ * used to identify the type of the message, defined as 'type.googleapis.com/full.type.name'.
+ * [value] is the serialized representation of the wrapped message. The [pack] and [unpack] methods
+ * are available to wrap and unwrap an `Any` message.
+ *
+ * Example
+ * ```
+ * let anyMessage = try AnyMessage.pack(person)
+ * let person = try anyMessage.unpack(Person.self)
+ * ```
+ */
+public struct AnyMessage {
+    public let typeUrl: String
+    public let value: Data
+
+    public init(typeUrl: String, value: Data) {
+        self.typeUrl = typeUrl
+        self.value = value
+    }
+
+    public enum DecodingError: Swift.Error, Equatable {
+        case typeUrlMismatch
+    }
+
+    public func unpack<Output: ProtoMessage & ProtoDecodable>(
```

Comment:
For the JVM, we also added `unpackOrNull` which would returns null in case nothing was found (instead of throwing).
Could we mirror it here? https://github.com/square/wire/blob/master/wire-library/wire-runtime/src/commonMain/kotlin/com/squareup/wire/AnyMessage.kt#L40-L49

Reply:
Yea I saw that one in the Kotlin file, but I think it's more idiomatic to achieve the same thing in Swift by simply using an optional try at the callsite - `let unpacked = try? anyMessage.unpack(something)`. If you feel strongly I can add that though